include/swift/AST/ASTContext.h

```diff
@@ -628,6 +629,13 @@ class ASTContext final {
     ArrayRef<SILParameterInfo> params, Optional<SILResultInfo> result,
     SILFunctionType::Representation trueRep);
 
+  /// Instantiates "Impl.Converter" if needed, then calls
+  /// ClangTypeConverter::getClangTemplateArguments.
```

Comment:
Updated. I'm going to leave the "Instantiates "Impl.Converter" if needed" bit so that people know they don't have to instantiate it before they call this member.